src/node.rs

```diff
@@ -619,7 +649,7 @@ impl<S: ForkSource + std::fmt::Debug> InMemoryNode<S> {
             InMemoryNodeInner {
                 current_timestamp: f.block_timestamp + 1,
                 current_batch: f.l1_block.0 + 1,
-                current_miniblock: f.l2_miniblock,
+                current_miniblock: f.l2_miniblock + 1,
```

Comment:
Are we sure this is correct after https://github.com/matter-labs/era-test-node/pull/102 ? I think we updated to have `current_miniblock` be the latest block instead of the next block per https://github.com/matter-labs/era-test-node/pull/102#discussion_r1321496043